docs/upgrade.html

```diff
@@ -47,14 +47,16 @@
 
 <p><b>Note:</b> Bumping the protocol version and restarting can be done any time after the brokers were upgraded. It does not have to be immediately after.
 
-<ol>
-    <li>Upgrading a Kafka Streams Applications:
-        <ul>
-            <li>You need to recompile your code. Just swapping the jar file will not work and will break your appliation.</li>
-            <li>If you use a custom timestamp extractor, you will need to update this code, because the <code>TimestampExtractor</code> interface was changed.</li>
-        </ul>
-    </li>
-</ol>
+<h5><a id="upgrade_1020_streams" href="#upgrade_1020_streams">Upgrading a Kafka Streams Application</a></h5>
+<ul>
+    <li> You need to recompile your code. Just swapping the jar file will not work and will break your application. </li>
+    <li> <code>KStreamBuilder#table()</code> always requires a store name. </li>
+    <li> <code>KTable#through()</code> always requires a store name. </li>
+    <li> If you use a custom timestamp extractor, you will need to update this code, because the <code>TimestampExtractor</code> interface was changed. </li>
```

Comment:
For (1) there is no changed -- we had only two options, and if user switches from non-default to `WallclockTimestampExtractor` there is nothing to update. Will add a link to streams.html 
"API changes"